src/iks003_incompressibleLinearElasticity.cpp

```
@@ -203,9 +207,9 @@ int main(int argc, char **argv) {
   if (ld.info() != Eigen::Success) assert(false && "Failed Solve");
```

This should throw an exception and should not be an assert.

---

modified